khmer/_oxli/graphs.pyx

```diff
@@ -281,17 +295,31 @@ cdef class QFCounttable(Hashtable):
         if type(self) is QFCounttable:
             self.c_table.reset(<CpHashtable*>new CpQFCounttable(k, int(log(starting_size, 2))))
 
+    @classmethod
+    def load(cls, file_name):
+        """Load the graph from the specified file."""
+        cdef Hashtable table = cls(1, 1)
+        deref(table.c_table).load(_bstring(file_name))
+        return table
 
-cdef class BigCountHashtable(Hashtable):
-    def set_use_bigcount(self, bigcount):
-        deref(self.c_table).set_use_bigcount(bigcount)
+cdef class Counttable(Hashtable):
+    def __cinit__(self, int k, int starting_size, int n_tables):
+        if type(self) is Counttable:
```

Comment:
yeah -- in cython, `__cinit__` is *always* called for every class in the inheritance hierarchy. you only want the tip of the hierarchy calling the pointer reset.